src/main/java/exceptions/DukeException.java

```diff
@@ -0,0 +1,62 @@
+package exceptions;
+
+import java.lang.StringBuilder;
+
+import java.io.File;
+
+public class DukeException {
+    private static int expectedArgs = 1;
+    public static void validate(boolean secondaryCheck, String command,
```

Comment:
Perhaps the parameter `secondaryCheck` can be renamed to sound like a boolean (e.g. isXXX, hasXXX)? This is in line with what the coding standard recommends.

Reply:
Apologies for any inconvenience caused as you were looking through.
The previous version was not revised according to Java Coding Standards. 
A new updated version, following the Java Coding Standards, has been released. Do take a look!